server/utils/utils.ts

```diff
@@ -119,7 +119,7 @@ const jsonDtToDateShort = (dt: string): string => {
 
 const jsonDtToDateWithDay = (dt: string): string => {
   const momentDate = moment(dt, 'DD/MM/YYYY HH:mm')
-  return momentDate.isValid() ? momentDate.format('dddd Do MMMM YYYY') : null
+  return momentDate.isValid() ? momentDate.format('dddd D MMMM YYYY') : null
```

Comment:
Design confirmed we should be displaying cardinal dates (eg 1 December 2024) rather than ordinal dates (eg 1st December 2024) across the service so altered the date formatting again here.